getting_started.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Getting Started with Overwatch
+---
+
+Overwatch is a security platform aimed at end users, it's purpose is to help keep track on your devices and servers preventing external access or detecting irregluar behavior that might otherwise go unnoticed.
+
+
+## Overwatch Devices
+
+Overwatch uses an installable agent to keep tabs on your devices, analyzing and logging it's regular behavior and enforcing the security rules that you configure on our web application. Devices refer to any platform in which our client agent can be installed, as such they are the main entity of focus in our web interface.
+
+## The Overwatch Agent
+
+Our agent runs as a service in your device, it's purpose it's two-fold: it enforces any [Security Policy rules](#security-policies) that you set up, and it sends regular status updates to the Overwatch servers for analysis.
+
+The stats sent out by the agent include plain keepalives, inbound and outbound data traffic, open connections and general details such as its IP. The privacy of our user's data is as a main concern at Overwatch: communications between the agent and the overwach server are encrypted, and while we do analyze the behaviour of the device - such as the amounts of traffic exchanged between the device and the network - the actual data is never sent down the wire. If you have any concerns about privacy issues, please [get in touch](get-in-touch) and we'll be happy to discuss any doubts.
```

Comment:
Besides privacy concern regarding traffic, I think it is also worth mentioning that our agent has to run with root privilege, which might mean a lot to certain user.